salt/cloud/clouds/msazure.py

```diff
@@ -2043,16 +2043,18 @@ def list_input_endpoints(kwargs=None, conn=None, call=None):
 
     ret = {}
     for item in data:
-        if 'Role' not in item:
-            continue
-        for role in item['Role']:
```

Comment:
This will contain RoleType since it's the first element in the dictionary and not the planned dictionary itself.